test/cli/dump-config-schema/args

```diff
@@ -0,0 +1,8 @@
+--dump-config
```

Comment:
I see no use-case for `dump-config-schema` test, because the field is not serialized to the JSON.

Reply:
Yes, but it is covered by the `simple-no-schema` test, right?

Reply:
Original `dump-config` test was testing the `--dump` flag, so I don't know if we need two tests for the same thing.

Reply:
:+1: Thanks for the explanation, so could we just give the test a better name?

`simple-no-schema` -> `mode-schema-empty-manifest`
`dump-config-schema` -> `mode-schema-missing-manifest`

Or something similar.

Reply:
(maybe this creates a need to rename other tests too)

Reply:
Thanks! :heavy_check_mark: